src/components/Results.tsx

```diff
@@ -35,7 +37,18 @@ const StyledContainer = styled(Box)(({ theme }) => ({
 }));
 
 export const Results: React.FC = () => {
-  const { accommodations, isFetching, latestQueryParams } = useAccommodationsAndOffers();
+  // to highlight a given event marker use url params "focusedEvent"
+  const [searchParams] = useSearchParams();
+  const focusedEvent = useMemo(
+    () => searchParams.get('focusedEvent') ?? '',
+    [searchParams]
+  );
+
+  // apply a callback function to transform returned accommodation objects
+  const { accommodations, isFetching, latestQueryParams } = useAccommodationsAndOffers(
```

Comment:
You also pas the `focusedEvent` in the `MapBox` component. 

Why is this done twice?

Reply:
`Results` and `MapBox` are 2 entirely independent components. They can't reuse this value between them